Assets/Scripts/Auction/AuctionDriver.cs

```diff
@@ -115,6 +125,70 @@ private IEnumerator WaitAndStartCameraAnimation()
     {
         yield return new WaitForSeconds(biddingBeginDelay);
         cameraAnimator.SetTrigger("start");
+        isAuctionStart = true;
+    }
+
+    [Server]
+    public void StopAuctionEarly()
+    {
+        biddingPlatforms.ToList().ForEach(platform => platform.ForceEndAuction());
+    }
+
+    public void AddYieldingPlayer(PlayerManager player)
+    {
+        if (yieldingPlayers.Contains(player))
+            return;
+
+        yieldingPlayers.Add(player);
+        OnYieldChange?.Invoke();
+
+
+        if (isAuctionStart && IsAuctionYielded())
+            StartCoroutine(nameof(WaitAndTryAuctionEnd));
```

Comment:
else stop the coroutine! 

currently it will
- wait for 3 seconds before checking if it should stop :skull: 
- show SKPPING IN X after going back to YIELD X/Y when someone leaves, until those 3 seconds have passed

Reply:
Fixed 👍